core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -283,32 +419,39 @@ describe('Overview', () => {
-            browser.wait(EC.visibilityOf(jazzServices_po.getLogs()), timeOutHigh);
-            jazzServices_po.getLogs().click();
-            browser.wait(EC.visibilityOf(jazzServices_po.getFilterIcon()), timeOutHigh);
+      it('Verify METRICS and Navigation to Swagger' , () => {
+            // Navigation to services
+            browser.wait(EC.visibilityOf(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getAwsServiceName()), timeOutHigh);
+            //To Navigate to the particular service and verifying the Page
+            jazzServices_po.getAwsServiceName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getServiceNameHeader()), timeOutHigh);
+            browser.sleep(45000);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getRefresh()), timeOutHigh);
+            jazzServices_po.getRefresh().click();
+            browser.sleep(30000);
+            jazzServices_po.getProdName().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.getProdHeader()), timeOutHigh);
+            browser.wait(EC.visibilityOf(jazzServices_po.getRefresh()), timeOutHigh);
             browser.driver.switchTo().activeElement();
-            jazzServices_po.getFilterIcon().click();
-            jazzServices_po.getDropDown().click();
-            jazzServices_po.getDay().click();
-            browser.sleep(2000);
-            jazzServices_po.getDropDown().click();
-            jazzServices_po.getWeek().click();
-            browser.sleep(2000);
-            expect(jazzServices_po.getWeekVerify().getText()).toEqual('WEEK');
-            jazzServices_po.getDropDown().click();
-            jazzServices_po.getMonth().click();
-            browser.sleep(2000);
-            expect(jazzServices_po.getMonthVerify().getText()).toEqual('MONTH');
-            jazzServices_po.getDropDown().click();
-            jazzServices_po.getYear().click();
-            expect(jazzServices_po.getYearVerify().getText()).toEqual('YEAR');
-            browser.sleep(2000);
+            browser.sleep(15000);
+            jazzServices_po.getMetrices().click();
+            browser.driver.sleep(5000);
+            browser.wait(EC.visibilityOf(jazzServices_po.getDeploymentStatus()), timeOutHigh);
+            jazzServices_po.getDeploymentStatus().click();
+            browser.wait(EC.elementToBeClickable(jazzServices_po.getWebsiteLink()), timeOutHigh);
+            jazzServices_po.getWebsiteLink().click();
+            browser.close();
+            browser.refresh();
+            jazzServices_po.getMetrices().click();
+            jazzServices_po.getMetricsChildOne().click();
+            jazzServices_po.getMetricsChildTwo().click();
+            jazzServices_po.getMetricsChildThree().click();
+            jazzServices_po.getMetricsChildFour().click();
+            jazzServices_po.getMetricsChildFive().click();
+            jazzServices_po.getMetricsChildSix().click();
```

Comment:
i guess you still need to add the assert?

Reply:
I have added the swagger success/Error code

Reply:
where?

Reply:
not really, as mentioned multiple times before every testcase(it block) is suppose to assert something.

Reply:
let me check in that case where i can add some assert,

Reply:
Added verification for website 'Get Start'